### PR TITLE
Sweets/smart wallet

### PR DESCRIPTION
**Pull request by sweetmantech**
# View Collection
- display contents of profile wallet on Profile page
 
Screenshot 2023-08-04 at 8 13 38 PM

### CODE REVIEW

File: components/ProfilePage/PreWalletCollection.tsx [hunk 1]
```diff
@@ -187,10 +157,18 @@ const PreWalletCollection = () => {
...
+            {ownedNfts.map((data, i) => (
               // eslint-disable-next-line react/no-array-index-key
               <div key={i} className="flex flex-col items-center gap-y-[5px]">
-                <div className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]" />
+                {data.image ? (
+                  <img
+                    className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]"
+                    src={data.image}
+                    alt={data.label}
+                  />
```
Comment (towardsnewtech) [inline components/ProfilePage/PreWalletCollection.tsx]: gonna change this part with next/image using shared/Media component 🛠️

File: components/ProfilePage/PreWalletCollection.tsx [hunk 2]
```diff
@@ -187,10 +157,18 @@ const PreWalletCollection = () => {
...
+                ) : (
+                  <div className="w-[30px] h-[30px] samsungS8:w-[35px] samsungS8:h-[35px] lg:w-[93px] lg:h-[93px] bg-white rounded-[5px] lg:rounded-[15px]" />
+                )}{" "}
```
Comment (towardsnewtech) [inline components/ProfilePage/PreWalletCollection.tsx]: {" "} => Can I remove this? 🚮

Comment (sweetmantech) [reply components/ProfilePage/PreWalletCollection.tsx]: should be fixed now 👍

Comment (sweetmantech) [reply components/ProfilePage/PreWalletCollection.tsx]: thank you, gonna leave this in your capable hands 🙇

**Pull request merged**